### PR TITLE
AV-44749: Added test case for default objects

### PR DESCRIPTION
**Pull request by grastogi23**

### CODE REVIEW

**Comment by grastogi23** [comment]
AV-44761 #commit

File: go/session/avisession_test.go [hunk 1]
```diff
@@ -187,6 +188,68 @@ func TestAviPool(t *testing.T) {
...
+	var npool3 models.Pool
+	err = avisess.Put("api/pool/"+npool2.UUID, npool2, &npool3)
+
+	if err != nil{
+		t.Errorf("Pool Patch failed %s", err)
+	}
+
+	// AV-44749: This logic should be flipped after fixing AV-44749.
+	if npool3.InlineHealthMonitor == false {
+		t.Errorf("Pool iniline health monitor setting changed to true")
```
Comment (ypraveen) [inline go/session/avisession_test.go]: typo -- iniline

File: go/session/avisession_test.go [hunk 2]
```diff
@@ -187,6 +188,68 @@ func TestAviPool(t *testing.T) {
...
+
+	var npool2 models.Pool
+	err = avisess.GetObjectByName("pool", pname, &npool2)
+
+	if err != nil {
+		t.Errorf("Pool %s lookup failed", pname)
+	}
+
+	if npool2.InlineHealthMonitor == false {
+		t.Errorf("Pool iniline health monitor setting changed")
```
Comment (ypraveen) [inline go/session/avisession_test.go]: typo here too

File: go/session/avisession_test.go [hunk 3]
```diff
@@ -187,6 +188,68 @@ func TestAviPool(t *testing.T) {
...
+	tpool.InlineHealthMonitor = true
+	var res models.Pool
+	err := avisess.Post("api/pool", tpool, &res)
+	glog.Infof("res: %s, err: %s", res, err)
+	if err != nil {
+		t.Errorf("Pool Creation failed: %s", err)
+	}
+
+	if res.InlineHealthMonitor == false {
+		t.Errorf("Pool iniline health monitor setting changed")
```
Comment (ypraveen) [inline go/session/avisession_test.go]: typo -- iniline

File: go/session/avisession_test.go [hunk 4]
```diff
@@ -12,6 +12,8 @@ import (
 
 var AVI_CONTROLLER = os.Getenv("AVI_CONTROLLER")
 var AVI_PASSWORD = os.Getenv("AVI_PASSWORD")
+var AVI_VERSION = os.Getenv("18.1.3")
```
Comment (ypraveen) [inline go/session/avisession_test.go]: Getenv("AVI_VERSION")?

**Comment by grastogi23** [comment]
@ypraveen added fix for review comments

**Pull request merged**